crates/ruff/src/rules/pylint/rules/yield_in_init.rs

```diff
@@ -9,6 +12,19 @@ use crate::{
 };
 
 define_violation!(
+    /// ### What it does
+    /// Checks for `__init__.py` methods that turned into generators
```

Comment:
It's `__init__`  methods ... not `__init__.py` methods (you made the mistake twice).